tests/test_helpers.py

```diff
@@ -1,10 +1,46 @@
 # -*- coding: utf-8 -*-
 
+import pytest
+
+from sceptre.exceptions import PathConversionError
 from sceptre.helpers import get_external_stack_name
+from sceptre.helpers import normalise_path
 
 
 class TestHelpers(object):
 
     def test_get_external_stack_name(self):
         result = get_external_stack_name("prj", "dev/ew1/jump-host")
         assert result == "prj-dev-ew1-jump-host"
+
+    def test_normalise_path_with_valid_path(self):
+        path = normalise_path("valid/path")
+        assert path == "valid/path"
```

Comment:
These assertions will fail on Windows (I assume - I don't have a Windows environment to test in). It would be OK I think to assert on `path.join`:

```py
assert path == join("valid", "path")
```